UnitTests/ReviewServiceTests.cs

```diff
@@ -183,7 +183,7 @@ public async Task GetAverage_GetTheAverageRatingOfTheGame()
             var average = await service.GetAverageRating(testGame.GameId);
             //Assert
             Assert.IsNotNull(average);
-            Assert.That(average, Is.EqualTo(3).Within(0.5));
+            Assert.That(average, Is.EqualTo(4).Within(0.5));
```

Comment:
This is weird...

Reply:
Yeah I'm not entirely sure why that was off but the new system doesn't accept 0 rating, but I think in the test it thinks it does, hence why it was off by 1